### DISCUSSION TITLE
Welcome to lazygit Discussions!

### DISCUSSION THREAD

**Discussion by jesseduffield** [jesseduffield/lazygit#1115] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jesseduffield** [comment]
I'm just using the default discussion intro text there ^ but I'm interested to see how a discussions section in the repo could help steer the direction of the project and get more people interested in it. To start with: I'm Jesse and I made lazygit a couple years ago now, and have been working with lazygit's fantastic team of collaborators ever since.

**Comment by thistlillo** [comment]
First of all thanks for your software, it is a real jewel, clearly implemented with love :-)

One question. Today, for the first time, the GUI has been stuck in "Pushing... <spinning bar>". The underlying process did not push anything and the spinning wheel kept rotating. I am working on an Ubuntu 20.04 with latest version of lazygit (0.24.2).

While it was stuck, I wondered whether it is possible to follow the output of the 'git push' command. I browsed through the menu and looked for it on the internet, but I could not find any info.

Is is possible to have the output of 'git' on screen? 

Thanks!
Cheers,

**Reply by jesseduffield** [reply]
Thanks for your kind words @thistlillo . Currently we don't have any way of showing a stream of git output. The reason is that our underlying library, gocui, doesn't know how to interpret certain ANSI escape codes that might be returned from git (mostly a problem when it comes to git hooks). There's a branch of the gocui library that I made for a separate project, lazynpm, but it would take some time to get that working for lazygit.

The closest thing you'll get is to run `lazygit --logs` and `lazygit --debug` in different tabs and take a look at the git commands that are run, but that's not quite the same as seeing the output of those commands.

**Comment by ryanpagel** [comment]
Is there a way to clear the Recent Repositories list or to selectively remove from the Recent Repos list certain repositories that are longer being used locally?  Thanks!

**Reply by jesseduffield** [reply]
First off: apologies for the late reply, github didn't send me notifications about comments in these discussion threads!

Currently there's no easy way to do it from within lazygit, but you can remove them manually from the `state.yml` file which lives in the same directory as the `config.yml` file. If you want to see the functionality added to lazygit itself, I would raise an issue for it :)
